solidity/random-beacon/hardhat.config.ts

```diff
@@ -0,0 +1,50 @@
+import { HardhatUserConfig } from "hardhat/config"
+
+import "@keep-network/hardhat-helpers"
+import "@keep-network/hardhat-local-networks-config"
+import "@nomiclabs/hardhat-waffle"
+import "@nomiclabs/hardhat-ethers"
+import "hardhat-deploy"
+import "@tenderly/hardhat-tenderly"
+
+const config: HardhatUserConfig = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.5",
+      },
+    ],
+  },
+  paths: {
+    artifacts: "./build",
+  },
+  networks: {
+    hardhat: {
+      forking: {
+        // forking is enabled only if FORKING_URL env is provided
+        enabled: !!process.env.FORKING_URL,
+        // URL should point to a node with archival data (Alchemy recommended)
+        url: process.env.FORKING_URL || "",
+        // latest block is taken if FORKING_BLOCK env is not provided
+        blockNumber: process.env.FORKING_BLOCK
+          ? parseInt(process.env.FORKING_BLOCK)
+          : undefined,
+      },
+      tags: ["local"],
+    },
+    ropsten: {
+      url: process.env.CHAIN_API_URL || "",
+      chainId: 3,
+      accounts: process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY
+        ? [process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY]
+        : undefined,
+      tags: ["tenderly"],
+    },
+  },
+  tenderly: {
```

Comment:
If we add `networks_TEMPLATE.ts` file it would be nice to have also an entry here:
```suggestion
  // // Define local networks configuration file path to load networks from the file.
  // localNetworksConfig: "./.hardhat/networks.ts",
  tenderly: {
```

Reply:
Added 168397b